### PR TITLE
Multi asset market price generator: Add BTC and ETH

### PR DESCRIPTION
**Pull request by ghost**
I added BTC and ETH price processes (naive / mock processes as done so far) for all market price models: `QUANTLIB`, `SCENARIO` with historical data and `SCENARIO` with mock data. The data pipeline is adjusted to incorporate historical BTC and ETH data. 

How to test: 
- Make sure https://github.com/celo-org/mento2-model/blob/main/model/system_parameters.py#L50 is `MarketPriceModel.SCENARIO`
- Alternate the data source between `mock` and `historical` in https://github.com/celo-org/mento2-model/blob/main/experiments/simulation_configuration.py#L12

The plots in the buy and sell notebook have been adjusted accordingly. 

What is not yet included is adding the assets to the reserve, I'll do this separately

### CODE REVIEW

**Review by bowd — CHANGES_REQUESTED** [review]
Small comments but looks good 👍

File: model/system_parameters.py [hunk 1]
```diff
@@ -62,25 +62,41 @@ class Parameters:
...
             }
         ]
     )
-    correlation: List[float] = default([[[1, 0, 0], [0, 1, 0], [0, 0, 1]]])
-    drift_market_price: List[float] = default([[-5*5, 0]])
+    correlation: List[float] = default([[[1, 0, 0, 0],
+                                         [0, 1, 0, 0],
+                                         [0, 0, 1, 0],
+                                         [0, 0, 0, 1]]])
+    drift_market_price: List[float] = default([[-5*5, 0, -5*4, -5*4]])
```
Comment (sissnad) [inline model/system_parameters.py:76]: This parameter is not required any longer

File: data/historical_market_data/market_data_pipeline.py [hunk 2]
```diff
@@ -22,6 +24,30 @@
...
+eth_usd = pd.read_csv(eth_path, usecols=['date', 'symbol', 'close'])
+eth_usd = eth_usd.rename(columns={'close': 'eth_usd'})
+eth_usd['date'] = pd.to_datetime(eth_usd['date'])
+eth_usd = eth_usd.set_index(['date'])
+eth_usd = eth_usd.sort_index()
+eth_usd = eth_usd[(eth_usd.index >= '2022-03-01') & (eth_usd.index <= '2022-03-11')]
+eth_usd = eth_usd.resample('5S').ffill().dropna()
+TEN_EPOCHS = 17280*10+1
+assert eth_usd.shape[0] == TEN_EPOCHS
+
```
[the remark this answers is not in the document]
Comment (sissnad) [reply data/historical_market_data/market_data_pipeline.py:48]: Already before I have been thinking, everything that happens in `market_data_pipeline` could live in `DataFeed`. Together with some formatting helper functions, e.g. a function that adds creates resamples from price files and adds them to the required array format for further consumption in scenario generation or historical simulation

[the remark this answers is not in the document]
Comment (sissnad) [reply data/historical_market_data/market_data_pipeline.py:48]: But to have us move on, this should probably be a new issue

**Comment by sissnad** [comment]
Looks good to me!

**Review by sissnad — APPROVED** [review]

File: data/historical_market_data/market_data_pipeline.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (tobikuhlmann) [reply data/historical_market_data/market_data_pipeline.py:48]: agreed, adding an issue

**Review by bowd — APPROVED** [review]

**Pull request merged**